eng/Versions.props

```diff
@@ -3,6 +3,7 @@
 <Project>
   <PropertyGroup>
     <VersionPrefix>17.7.0</VersionPrefix>
+    <DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

Comment:
Nit: we usually like to put this _on the same line as VersionPrefix_ so that it is guaranteed to cause a merge conflict when moving things around. But not a big deal!

Reply:
aaah - that's why